etcdserver/api/v3rpc/rpctypes/error.go

```diff
@@ -22,6 +22,7 @@ import (
 var (
 	// server-side error
 	ErrGRPCEmptyKey     = grpc.Errorf(codes.InvalidArgument, "etcdserver: key is not provided")
+	ErrGRPCValue        = grpc.Errorf(codes.InvalidArgument, "etcdserver: value is provided")
```

Comment:
probably we can simply ignore the value instead of returning error?

/cc @heyitsanthony opinion?